src/encoded/genomic_data_service.py

```diff
@@ -0,0 +1,136 @@
+import requests
+
+REGISTRY_DATA_SERVICE = 'genomic_data_service'
+RNA_GET_FILTERS = ['assayType', 'annotation']
+RNA_GET_ENDPOINT = '/expressions/bytes'
+
+# react component orders columns by "the position" in the hash map
+RNA_GET_COLUMNS = {
+    'featureID': {'title': 'Feature ID'},
+    'tpm': {'title': 'Counts (TPM)'},
+    'fpkm': {'title': 'Counts (FPKM)'},
+    'assayType': {'title': 'Assay (RNA SubType)'},
+    'libraryPrepProtocol': {'title': 'Experiment'},
+    'expressionID': {'title': 'File'},
+    'annotation': {'title': 'Genome Annotation'}
+}
+
+class GenomicDataService():
+    # default search value is a temporary feature for the current alpha UI client
+    def __init__(self, registry, request, default_search='ENSG00000088320.3'):
+        self.path = registry.settings.get(REGISTRY_DATA_SERVICE)
+        self.request = request
+        self.default_search = default_search
+
+        self.parse_params()
+
+
+    def parse_params(self):
+        params = self.request.params
+
+        self.genes   = params.get('genes')
+        self.units   = params.get('units', 'tpm')
+        self.page    = params.get('page', 1)
+        self.sort    = params.get('sort')
```

Comment:
Nit! https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

Reply:
that was me still with a Rails brain haha